integrations/cohere/tests/test_cohere_chat_generator.py

```diff
@@ -7,6 +7,7 @@
 from haystack.components.generators.utils import print_streaming_chunk
 from haystack.dataclasses import ChatMessage, ChatRole, StreamingChunk
 from haystack.utils import Secret
+
```

Comment:
Was this intentional? It added extra files in PR.

Reply:
yeah, related to linting...